shell/platform/tizen/flutter_tizen_engine_test.cc

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/tizen/flutter_tizen_engine.h"
+#include "gtest/gtest.h"
+
+TEST(FlutterTizenEngineTestSimple, Create_Headless) {
+  flutter::FlutterTizenEngine* tizen_engine =
+      new flutter::FlutterTizenEngine(false);
+  EXPECT_TRUE(tizen_engine != nullptr);
+  delete tizen_engine;
+}
+
+// TODO
+TEST(FlutterTizenEngineTestSimple, DISABLED_Create_Headed) {
+  flutter::FlutterTizenEngine* tizen_engine =
+      new flutter::FlutterTizenEngine(true);
+  EXPECT_TRUE(tizen_engine != nullptr);
+  delete tizen_engine;
+}
+
+class FlutterTizenEngineTest : public testing::Test {
+ public:
+  FlutterTizenEngineTest() {
+    std::string tpk_root;
+    char path[256];
+    EXPECT_TRUE(getcwd(path, sizeof(path)) != NULL);
+    tpk_root = path + std::string("/tpkroot");
+
+    assets_path_ = tpk_root + "/res/flutter_assets";
+    icu_data_path_ = tpk_root + "/res/icudtl.dat";
+    aot_lib_path_ = tpk_root + "/lib/libapp.so";
+
+    switches_.push_back("--disable-observatory");
+    // switches_.push_back("--verbose-logging");
+    // switches_.push_back("--enable-dart-profiling");
+    // switches_.push_back("--enable-checked-mode");
+  }
+
+ protected:
+  void SetUp() {
+    engine_prop_.assets_path = assets_path_.c_str();
+    engine_prop_.icu_data_path = icu_data_path_.c_str();
+    engine_prop_.aot_library_path = aot_lib_path_.c_str();
+    engine_prop_.switches = switches_.data();
+    engine_prop_.switches_count = switches_.size();
+
+    auto engine = std::make_unique<flutter::FlutterTizenEngine>(false);
```

Comment:
I'll polish this part up when https://github.com/flutter-tizen/engine/pull/133 is merged.

Reply:
Thank you I'll look forward to it. :eyes: